arrows/core/transform_detected_object_set.cxx

```diff
@@ -213,23 +213,24 @@ box_around_box3d( const kwiver::vital::camera_perspective_sptr camera,
     projected_points.row(i) = camera->project(box3d.row(i)).transpose();
   }
 
-  vector_4d out_box;
-  out_box << (vector_2d)projected_points.colwise().minCoeff(),
-             (vector_2d)projected_points.colwise().maxCoeff();
+  auto out_box = vital::bounding_box<double>(
+    (vector_2d)projected_points.colwise().minCoeff(),
+    (vector_2d)projected_points.colwise().maxCoeff());
 
   return out_box;
 }
 
 // ---------------------------------------------------------------------------
-vector_4d
+vital::bounding_box<double>
 transform_detected_object_set::
 view_to_view( const kwiver::vital::camera_perspective_sptr src_camera,
               const kwiver::vital::camera_perspective_sptr dest_camera,
-              const vector_4d bounds ) const
+              const vital::bounding_box<double> bbox ) const
 {
   Eigen::Matrix<double, 8, 3> box3d =
-    this->backproject_bbox( src_camera, bounds );
-  vector_4d tgt_box = this->box_around_box3d( dest_camera, box3d );
+    this->backproject_bbox( src_camera, bbox );
+  vital::bounding_box<double> tgt_box =
```

Comment:
Here is another case where you can eliminate the temporary variable a directly return the result of the function call.